### ISSUE TITLE
Update prompt no longer working

### ISSUE THREAD

**Issue by samajammin** [netlify/cli#1492]
**Describe the bug**

When killing my `netlify dev` server, I used to receive a prompt when updates where available & the appropriate cli command to run. Now when I kill my server, I get this output:

```
   ╭──────────────────────────────────────────────────────────╮
   │                                                          │
   │   Update available {currentVersion} → {latestVersion}    │
   │              Run {updateCommand} to update               │
   │                                                          │
   ╰──────────────────────────────────────────────────────────╯

```

It appears the string interpolation is broken?

**To Reproduce**

1. Run `netlify dev`
2. Kill server (e.g. `control` + `C`)
3. View output

**Configuration**

```
$ npx envinfo --system --binaries --npmPackages netlify-cli --npmGlobalPackages netlify-cli

  System:
    OS: macOS 10.15.5
    CPU: (8) x64 Intel(R) Core(TM) i7-4980HQ CPU @ 2.80GHz
    Memory: 361.68 MB / 16.00 GB
    Shell: 5.0.7 - /usr/local/bin/bash
  Binaries:
    Node: 10.19.0 - ~/.nvm/versions/node/v10.19.0/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.14.8 - ~/.nvm/versions/node/v10.19.0/bin/npm
  npmGlobalPackages:
    netlify-cli: 2.65.7

```

**Expected behavior**

Output should list the actual command to run.

**CLI Output**

See above.

**Additional context**

None.

**Comment by samajammin** [comment]
This appears to be working again (as of `2.68.8`):
```
   ╭────────────────────────────────────────╮
   │                                        │
   │   Update available 2.68.8 → 2.69.0     │
   │   Run npm i -g netlify-cli to update   │
   │                                        │
   ╰────────────────────────────────────────╯
```

**Comment by lukebennett88** [comment]
I've been having this issue for a while. Not sure if it's an issue with `netlify-cli` or `yarn`.

I've tried uninstalling and reinstalling, but nothing seems to work. Every time there is a new update I see this prompt in my terminal:

```
   ╭──────────────────────────────────────────────────────────╮
   │                                                          │
   │   Update available {currentVersion} → {latestVersion}    │
   │              Run {updateCommand} to update               │
   │                                                          │
   ╰──────────────────────────────────────────────────────────╯
   ```
